### DISCUSSION TITLE
problem with drag and drop component

### DISCUSSION THREAD

**Discussion by Salisyal** [LiveDuo/destack#66] [Q&A]
![image](https://user-images.githubusercontent.com/54141575/211494797-ed3161b2-b623-4bf0-b573-ec6ce3cebcbc.png)

Hello, my name is Faisyal, I want to try one of these open-source tools, there is a problem when I drag a component or section it changes to images and it doesn't match the component we are dragging.

**Comment by LiveDuo** [comment]
@ankitpanwar070798 Is this the problem you are facing?

**Comment by LiveDuo** [comment]
Hey @Salisyal, there's another report about this. Have you started off the "starter project" or used an existing project of yours?

PS: In the future it's better to use the "Issues" section for problems like this.
